### ISSUE TITLE
Class 'pelock\antispamform\AntiSpamForm' not found

### ISSUE THREAD

**Issue by evaldemar** [PELock/yii2-anti-spam-form#1]
Install with composer. Change ActiveForm to AntiSpamForm.

**Comment by eweitzel-bo** [comment]
I see the same behavior. Is there a workaround?

I use yii2.
I added pelock/yii2-anti-spam-form:* to the composer.json
I changed  ActiveForm to AntiSpamForm
I get the error Class 'pelock\antispamform\AntiSpamForm' not found

**Comment by PELock** [comment]
Did you execute `composer update`?

**Comment by eweitzel-bo** [comment]
Yes, i executed composer update and i can see that the extension is installed.

Von: Bartosz Wójcik ***@***.***>
Gesendet: Donnerstag, 16. Mai 2024 20:44
An: PELock/yii2-anti-spam-form ***@***.***>
Cc: Weitzel, Edgar ***@***.***>; Comment ***@***.***>
Betreff: Re: [PELock/yii2-anti-spam-form] Class 'pelock\antispamform\AntiSpamForm' not found (Issue #1)


Did you execute composer update?
